dockerfiles/Dockerfile.archlinux

```diff
@@ -1,23 +1,19 @@
-FROM archlinux/base
+FROM scratch
 MAINTAINER Iterait a.s. <docker@iterait.com>
 
+COPY --from=archlinux/base / /
```

Comment:
Interesting - why not just start from `archlinux/base` in the first place instead of copying the whole root hierarchy?

Reply:
Because `--squash` doesn't squash the base image, so we wouldn't really save anything.

Reply:
Of course... Thank you.